app/src/renderer/vuex/modules/user.js

```diff
@@ -27,6 +27,10 @@ export default ({ node }) => {
     },
     addHistory(state, path) {
       state.history.push(path)
+      window.analytics &&
+        window.analytics.send("pageview", {
```

Comment:
what else could we be tracking that we aren't?

Reply:
Let's move this here: https://github.com/cosmos/voyager/issues/1032